static/src/javascripts/projects/common/modules/commercial/acquisitions-ophan.js

```diff
@@ -62,12 +64,21 @@ export const addTrackingCodesToUrl = ({
         abTest,
     });
 
-    const params = {
+    const params: {
+        REFPVID: string,
+        INTCMP?: string,
+        acquisitionData: string,
+        bundle?: string,
+    } = {
         REFPVID: config.get('ophan.pageViewId') || 'not_found',
         INTCMP: campaignCode,
         acquisitionData: JSON.stringify(acquisitionData),
     };
 
+    if (geolocation === 'GB' && componentType === 'ACQUISITIONS_EPIC') {
```

Comment:
Since this isn't adding tracking codes to the URL, I'd rather it didn't go in `addTrackingCodesToUrl`